index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+<header>
+    <h1>Hi, I'm Jaime and I'm a runner!</h1>
+</header>
+    <main>
+        <section>
+            <h2>I never ran or played sports growing up, but as an adult I've ran:</h2>
+            <ul>
+                <li>13 half marathons</li>
+                <li> 2 full marathons</li>
+                <li> 1 ultra marathon</li>
+            </ul>
+            <img src= "./assets/canyon-de-chelly.jpeg">
+        </section>
+
+        <section>
+            <h2>Some highlights include:</h2>
+            <ul>
+                <li><strong>2009 -</strong> My first 5K was the Race For The Cure in Kansas City with my younger sister.</li>
+                <li><strong>2010 -</strong> My dad ran his first 5K and beat both me and my sister.</li>
+                <li><strong>2015 -</strong> My dad, my sister, and I all ran a half marathon together.</li>
+            </ul>
+            <img src="./assets/running-with-the-cows.jpg">
+        </section>
+
+        <section>
+           <h2>My goals are to run the world's largest races at these distances:</h2> 
+           <ol>
+               <li><strong>Half Marathon -</strong> I completed the world's largest in Gothenburg, Sweden in 2018. (check)</li>
+               <li><strong>Marathon - </strong>I've entered the drawing for the New York City marathon and didn't get in, but I haven't given up for 2020 yet! </li>
+               <li><strong>Ultramarathon -</strong> The largest (and oldest) ultra is Comrades which goes from Durban to Pietermaritzburg in South Africa.</li>
+               <img src="./assets/gothenburg.jpg">
+           </ol>
+        </section>
+        
+        <section>
+            <h3>Take the Jaime Quiz!</h3>
+            <button id='quiz-button'>Take Quiz</button>
+            <p id= "quiz-result">Take the quiz and get a score!</p>
+        </section>
+    </main>
+
+    <script type="module" src="app.js"></script>
+</body>
+
+</html>
```

Comment:
Eslint wants you to add a return after the last line of your code. That's what this error is about.